contracts/script/SP1ICS07Tendermint.s.sol

```diff
@@ -61,13 +63,15 @@ contract SP1TendermintScript is Script, IICS07TendermintMsgs {
         bytes32 updateClientVkey = json.readBytes32(".updateClientVkey");
         bytes32 membershipVkey = json.readBytes32(".membershipVkey");
         bytes32 ucAndMembershipVkey = json.readBytes32(".ucAndMembershipVkey");
+        bytes32 misbehaviourVkey = json.readBytes32(".misbehaviourVkey");
```

Comment:
Since this is a new program, it means we need a new program key to verify it in the proof verification process in Solidity.